### PR TITLE
Revisit tokenaccounts 

### PR DESCRIPTION
**Pull request by BouweCeunen**

### CODE REVIEW

File: src/store/api.ts [hunk 1]
```diff
@@ -418,35 +428,32 @@ export const createDeal = multiAsync(
 
 export const activateDeal = multiAsync(async (connection: Connection, wallet: Wallet) => {
 	const program = constructProgram(connection, wallet);
-	const _userUSDCTokenAccount = getUserUSDCTokenAccount(connection, wallet);
+	const _userAssociatedUSDCTokenAddressPK = getAssociatedUSDCTokenAddressPK(connection, wallet);
```
Comment (MaximPiessen) [inline src/store/api.ts:411]: This is not correct. We should be able to get the associated token address for an arbitrary borrower defined in the deal, not only the connected wallet.

Comment (BouweCeunen) [reply src/store/api.ts:411]: https://www.notion.so/credix/Allow-for-inputting-the-key-of-a-random-borrower-when-creating-a-deal-596483f76015436cadebbc242f46daa8 will be handled here

File: src/store/api.ts [hunk 2]
```diff
@@ -209,6 +209,16 @@ const getTreasuryPoolTokenAccountPK = multiAsync(async (connection: Connection,
 	return globalMarketStateData.treasuryPoolTokenAccount;
 });
 
+const getAssociatedUSDCTokenAddressPK = multiAsync(async (connection: Connection, wallet: Wallet) => {
```
Comment (RuvendeGroote) [inline src/store/api.ts:200]: remove `getUserUSDCTokenAccount` as it's no longer needed now that we use the associated token account.

**Review by RuvendeGroote — CHANGES_REQUESTED** [review]

**Comment by BouweCeunen** [comment]
to many merge conflicts, branched clean again from signing-authority https://github.com/credix-finance/credix-app/pull/18

**Pull request closed without merging**